src/pods/test-sentences/components/body.component.tsx

```diff
@@ -41,17 +53,23 @@ export const BodyComponent: React.FunctionComponent<Props> = props => {
             size="large"
             aria-label="contained primary button group"
           >
-            <Button>{present}</Button>
-            <Button>{past}</Button>
-            <Button>{participle}</Button>
+            <Button onClick={handleButtonValue} value="Present">
+              {present}
+            </Button>
+            <Button onClick={handleButtonValue} value="Past">
+              {past}
+            </Button>
+            <Button onClick={handleButtonValue} value="Participle">
+              {participle}
+            </Button>
           </ButtonGroup>
         </div>
```

Comment:
Enclose all this Button Group in a component, by doing this you could get this component as simple as:

```tsx
  return (
    <main className={mainContainer}>
      <h1 className={title}>{translation.toUpperCase()}</h1>
      <div className={backContainer}>
        <div className={pictureContainer}>
          <img className={picture} src={`/assets/verb-images/buy.png`} />
        </div>
        <div className={buttonGroupContainer}>
           <TensesButtonGroup/>
        </div>
        <div>
          <SentenceComponent
            sentenceSelected={sentenceSelected}
            rightAnswerValue={rightAnswerValue}
            verbForms={verbForms}
          />
        </div>
      </div>
    </main>
```

Tenses buttonGroup will have the proper callbacks and props